controllers/kubevirtmachine_controller_test.go

```diff
@@ -25,11 +25,26 @@ import (
 	"github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
 	"github.com/pkg/errors"
+	
+	
+	
+	
+	
+	
+	
```

Comment:
Please run `go fmt` to remove all those empty lines

Reply:
> Please run `go fmt` to remove all those empty lines

please run `mae sanity` - it will do that + other things.

Reply:
Fixed